### PR TITLE
docs(deployment): add deploy as a service in swarm

### PR DESCRIPTION
**Pull request by bvis**
I've added a new paragraph to launch Portainer as a service in the swarm cluster itself. It can be very useful.

### CODE REVIEW

File: docs/source/deployment.rst [hunk 1]
```diff
@@ -74,6 +74,17 @@ For example, when connecting to a remote Swarm node:
...
+You can deploy it as a service in the swarm cluster:
+
+.. code-block:: bash
+
+  $ docker service create \
+      --name portainer \
+      --constraint 'node.role == manager' \
+      --mount type=bind,src=/var/run/docker.sock,dst=/var/run/docker.sock \
+      portainer/portainer \
+      -H unix:///var/run/docker.sock \
```
Comment (deviantony) [inline docs/source/deployment.rst]: `-H unix:///var/run/docker.sock` is using the default value for the `-H` flag, you do not have to specify it. Please remove it.

File: docs/source/deployment.rst [hunk 2]
```diff
@@ -74,6 +74,17 @@ For example, when connecting to a remote Swarm node:
 
   $ docker run -d -p 9000:9000 portainer/portainer -H tcp://<REMOTE_HOST>:<REMOTE_PORT> --swarm
 
+You can deploy it as a service in the swarm cluster:
```
Comment (deviantony) [inline docs/source/deployment.rst]: Please update this text to: 

"If you're using swarm mode, you can also deploy it as a service in your cluster:"

File: docs/source/deployment.rst (hunk 1 above)
Comment (deviantony) [inline docs/source/deployment.rst]: That's a good thing to add in the docs overall, but maybe you should also add some information on how to access it in your cluster after that?

**Comment by deviantony** [comment]
@bvis 
That's a good thing to add in the docs overall, but maybe you should also add some information on how to access it in your cluster after that?

**Comment by bvis** [comment]
I've exposed a port to access in the same way as in the other examples.

**Comment by deviantony** [comment]
Thanks @bvis !

**Pull request merged**